aggie-coding-club/Rev-Registration
# Move include full section into course card

## Description

This moves the include full sections toggle into the course cards and eliminates the configure card

## Rationale

This provides more control to the user when deciding which courses to include full sections for. It also removes the configure card because there is no longer any reason for it to exist.

## How to test

I didn't make any new tests for this because I thought it should be covered by the existing tests. You can also select a course(s) with restrictions such that a schedule can only be generated if full sections are included and try to generate a schedule. If it only returns schedules when the include full is included for those courses, it should be working.

## Screenshots



|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/16074716/102032866-8cf5ec00-3d7f-11eb-92df-b846c6d6207c.png)|![image](https://user-images.githubusercontent.com/16074716/102033052-0f7eab80-3d80-11eb-87be-6c97b9d7cbae.png)|

## Related tasks

Closing tag. Don't forget to link the issue w/ the PR in ZenHub at the bottom.

Closes #393 #354
